src/Amadevus.RecordGenerator.Attributes/RecordAttribute.cs

```diff
@@ -2,9 +2,40 @@
 using System;
 using System.Diagnostics;
 
-[AttributeUsage(AttributeTargets.Class, Inherited = false, AllowMultiple = false)]
-[CodeGenerationAttribute("Amadevus.RecordGenerator.Generators.RecordGenerator, Amadevus.RecordGenerator.Generators")]
-[Conditional("CodeGeneration")]
-public sealed class RecordAttribute : Attribute
+namespace Amadevus.RecordGenerator
 {
-}
+    /// <summary>
+    /// This attribute triggers record code generation. <see href="https://amis92.github.io/RecordGenerator/"/>
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class, Inherited = false, AllowMultiple = false)]
+    [CodeGenerationAttribute("Amadevus.RecordGenerator.Generators.RecordGenerator, Amadevus.RecordGenerator.Generators")]
+    [Conditional("CodeGeneration")]
+    public sealed class RecordAttribute : Attribute
+    {
+        /// <summary>
+        /// For the annotated type, generate
+        /// <see cref="DefaultRecordFeaturesAttribute.AssemblyDefaultFeatures"/>
+        /// or if not specified, <see cref="Features.Default"/>.
+        /// </summary>
+        public RecordAttribute()
```

Comment:
Would'nt it be easier to specify `Features.Default` as default value?
This way, you wouldn't have to check if a feature has been specified when calling `ConstructorArguments[0]`

Reply:
It'd not replace the check, because I'm only getting syntax-level values, and default parameter value is not taken into account here.

Also, the difference (whether it was specified or not) is what triggers the fallback to assembly-default. :)

Reply:
Oh yea, alright that makes sense 😄